plugins/dkim_sign.js

```diff
@@ -15,7 +15,9 @@ class DKIMSignStream extends Stream {
         super();
 
         this.selector = props.selector;
-        this.domain = props.domain;
+
+        // fix issue #2668 renaming reserved kw/property of 'domain' to 'domain_name'
+        this.domain_name = props.domain_name || props.domain;
```

Comment:
```suggestion
        this.domain_name = props.domain;
```

We only want to change the property name, not the calling property object.